docs/architecture/blazor-for-web-forms-developers/hosting-models.md

```diff
@@ -1,9 +1,80 @@
 ---
-title: Hosting models
+title: Blazor app hosting models
 description: Different ways to host your Blazor apps including in the browser on WebAssembly or on the server.
 author: danroth27
 ms.author: daroth
 ms.date: 09/11/2019
 ---
 
-# Hosting models
+# Blazor app hosting models
+
+Blazor apps can be hosted in IIS just like ASP.NET Web Forms apps. But Blazor apps can be hosted in other ways as well. When considering how to host your Blazor app, you have the option of hosting it client-side in the browser on WebAssembly or server-side in an ASP.NET Core app. 
+
+## Blazor WebAssembly apps
+
+Blazor WebAssembly apps execute directly in the browser on a WebAssembly based .NET runtime. Blazor WebAssembly apps function in a very similar way to front-end JavaScript frameworks like Angular or React, but instead of writing JavaScript you write C#. The .NET runtime is download with the app along with the app assembly and any required dependencies. No browser plugins or extensions are required. 
```

Comment:
```suggestion
Blazor WebAssembly apps execute directly in the browser on a WebAssembly-based .NET runtime. Blazor WebAssembly apps function in a very similar way to front-end JavaScript frameworks like Angular or React, but instead of writing JavaScript you write C#. The .NET runtime is downloaded with the app along with the app assembly and any required dependencies. No browser plugins or extensions are required. 
```